src/components/CMessageTable.vue

```diff
@@ -107,7 +96,7 @@ export default {
   },
 
   computed: {
-    ...mapState(['messages', 'refreshInterval', 'isLoading']),
+    ...mapState(['messages', 'refreshInterval', 'actors', 'isLoading']),
```

Comment:
What i would prefer, is for you to populate message with the actor.

for example there would be a getter in the state messageWithActors.

The populate the message wit the actor, build an object with actorName as key and the actor as value it will be more efficient o(N) where your solution is o(N2)

then in the table you will access `m.actor ? m.actor.queueName : ''`

Reply:
@antoinerabany I implemented one that uses O(1) in memory and time,  avoids O(n) in memory. Let me know what you think